### ISSUE TITLE
German Localization; Change Menu item translation of „File” to „Ablage”

### ISSUE THREAD

**Issue by nodevnul** [frescobaldi/frescobaldi#1668]
Hi.

I would like to suggest to change the translation of the menu-item „File” from „Datei” to „Ablage”.

The correct translation of the term „File” within the functional context in this application is „Ablage”. Unfortunately the english term „File” denotes several circumstances in information technology, whereas German distinguishes between „Datei” and „Ablage” („file“ and „filing”).

The german term „Datei” is defined as:

„Eine Datei ist in der Informationstechnologiedie Zusammenfassung gleichartiger digitaler Daten, die zum Speichern auf Datenträgern oder Speichermedien, zur Wiedergabe, zum Bearbeiten und zur Datenübertragung dient und durch einen Dateinamen identifiziert wird.”

Basically this means, that the term „Datei” means a set of data of equal kind within a file that is identified by a file-name. Whereas „Ablage” means the procedure of „filing” one or more files into a location.

I hope that the problem is described clearly. Now, some of you might argue that so many software - even MS, uses the term „Datei”. Well, unfortunately, they are all wrong.

Thanks for listening.

**Comment by 19joho66** [comment]
Every German applications uses "Datei". They might be wrong, but it has been a convention for decades. So using "Ablage" would not be expected by any user and would lead to irritations.
It just doesn't make sense that only Frescobaldi would use "Ablage" instead of "Datei"

**Comment by nodevnul** [comment]
> Every German applications uses "Datei". 

False.

macOS since the beginning
Things
iMessage
MacGiro
Sparkle
GraphicConverter
Threema
VLC Media Player (wow)
TexShop
LaTexIt
Texifier (Techpad Pro)
Cyberduck
Handbrake
MonkeyOffice


<but it has been a convention for decades.

False. Declaring lazyness as a convention over time is indeed very steep…

<So using "Ablage" would not be expected by any user and would lead to irritations.

False. Correct translations are a 100% expected by (educated) users. They do not lead to irritations but to delightment that also the small little things have been done correctly.

I mean really re-read your sentence: Doing a correct translation leads to irritation with native speakers… 

< It just doesn't make sense that only Frescobaldi would use "Ablage" instead of "Datei"

Nope, as shown…


Thanks for listening.

**Comment by papioara** [comment]
@nodevnul
You put an interesting question, and I think that your proposal is not made without good reasons. It made me rethink the question: "What is more appropriate: 'Datei' or 'Ablage'?"
I also did some research - first on my own computer (Linux Mint) and then on the web. It turns out that maybe semantically the former is more 'correct', but that statistically, although being maybe less accurate, the latter is much more frequently used. I have sifted through my own applications (well, not through **all** of them, but through about 20), and the ones with a classical menu bar all had 'Datei' in it. I am not too familiar with Windows, but from what I remember using it every now and then, it is exactly the same.
I do remember from my own computing history (starting with a donated Mac Performa 630CD) that on Mac OS, 'Ablage' is rather the standard menu item (found this confirmed on the internet). I have left the Apple universe long ago (when they switched to Intel CPUs, and I did not want to buy a new Mac so soon), so my own memories have become somewhat dim ;-)
I assume (also from your application list) that you are a Mac user who - no wonder - is used to the Mac OS conventions (and therefore 'Datei' strikes you much more than users of other OS'es).
Taking into account the conventions/habits (one might even call some of them not being completely correct strictly speaking) in the different operating system environments, I would suggest to take a pragmatic approach: Since everyone knows what to expect when they see 'Datei', we rather should leave it like that in spite of the fact that it is less accurate to semantic definition). Even Mac users in Boot Camp running Windows have to live with that ;-) It's not the end of the world. We should not be too rigid about these matters, unless they cause confusion. I cannot see this here. It's a bit like two signs in German: one says: "STOP", and the other one says "HALT". The words are different, but basically they want to tell you the same...

**Comment by papioara** [comment]
@jeanas 
I have the impression that you misunderstood @nodevnul (if I understood him correctly) assuming he had suggested a sweeping replacement of 'Datei' with 'Ablage'.
We have a German-specific problem here which at least for English and French ('File' and 'Fichier') does not exist. The suggestion was only about the top menu entry 'Datei' in the menu bar and nothing else. On Mac OS (which I have been using for quite a while before switching to Linux) however, you have always had 'Ablage' instead of 'Datei' in the German localization - see the screenshot I just took from my 1996 Mac which is still running.


![Auswahl_069](https://github.com/frescobaldi/frescobaldi/assets/31970041/ed0d155e-345d-4665-bebf-2a12e931590e)
![IMG_20230913_113054](https://github.com/frescobaldi/frescobaldi/assets/31970041/22f53a39-a7ee-4a61-92a1-910de2b6b8d9)
And as for the rest: see my last comment...
